gapic/src/main/com/google/gapid/server/GapisProcess.java

```diff
@@ -96,7 +96,7 @@ protected Exception prepare(ProcessBuilder pb) {
       args.add("-log-level");
       args.add(Logging.getGapisLogLevel());
 
-      gapirFlags = "--log " + new File(logDir, "gapir.log").getAbsolutePath() +
+      gapirFlags = "--log " + new File(logDir, "gapir.log").getAbsolutePath().replace("\\", "\\\\") +
```

Comment:
Ah, this is getting escaped by `text.SplitArgs`. It took me a while to figure out why this was needed.